### ISSUE TITLE
Password list path ? 

### ISSUE THREAD

**Issue by Bhavy11111** [MrHacker-X/BruteX#2]
It asks password list path after using custom password, what's that ?

**Comment by ladilla88** [comment]
Password listo parh

**Comment by BotKi11eR** [comment]
You need to give the path  where your custom password file is .  Like this : 
/data/data/com.termux/files/home/pass.txt ......
that file should contaij custom passwords you want to try .. 
go to the dirextory where your password dile is, and tupe 
pwd
you will get the path adress, simply copy it and paste it where it ask you for path, and write the fike name at the end of the psth .. Like in the example above.

**Comment by elliot-blip** [comment]
> You need to give the path where your custom password file is . Like this : /data/data/com.termux/files/home/pass.txt ...... that file should contaij custom passwords you want to try .. go to the dirextory where your password dile is, and tupe pwd you will get the path adress, simply copy it and paste it where it ask you for path, and write the fike name at the end of the psth .. Like in the example above.

what do i type into termux

**Comment by BotKi11eR** [comment]
> > You need to give the path where your custom password file is . Like this : /data/data/com.termux/files/home/pass.txt ...... that file should contaij custom passwords you want to try .. go to the dirextory where your password dile is, and tupe pwd you will get the path adress, simply copy it and paste it where it ask you for path, and write the fike name at the end of the psth .. Like in the example above.
> 
> what do i type into termux

Bro here I wrote what you need to write there ..

**Comment by BotKi11eR** [comment]
Which part exactly you didn't get?

**Comment by EsxRetro025** [comment]
i got that Same Problem. and i need Your Help, What Should write To See Directory Path?

**Comment by shivamsingh915** [comment]
What should I type after =passWord list path

**Comment by BotKi11eR** [comment]
> i got that Same Problem. and i need Your Help, What Should write To See Directory Path?

pwd

**Comment by BotKi11eR** [comment]
> What should I type after =passWord list path 

The directory path where your custom password list is placed..

**Comment by sarla45516** [comment]
![Uploading IMG_20221102_083023.jpg…]()
 Why it showing this

**Comment by BotKi11eR** [comment]
> ![Uploading IMG_20221102_083023.jpg…]()
>  Why it showing this

Your picture is not uploaded correctly I guess .. try posting again, its not opening

**Comment by Ak40999** [comment]
![Screenshot_2022-12-09-10-38-05-93_84d3000e3f4017145260f7618db1d683.jpg](https://user-images.githubusercontent.com/120032131/206625154-1b4ebec3-c37a-4582-ac75-99ff935aff13.jpg)

Password list path ke

**Comment by BotKi11eR** [comment]
> ![Screenshot_2022-12-09-10-38-05-93_84d3000e3f4017145260f7618db1d683.jpg](https://user-images.githubusercontent.com/120032131/206625154-1b4ebec3-c37a-4582-ac75-99ff935aff13.jpg)
> 
> Password list path ke 

Bro do you know what "Custom" means? You are supposed to have a txt file with some custom passwords you want to try on that account . And you need to give the location of that file in here . Go to the directory where your custom passwords file is . And type without brackets ( pwd ) . It will give you the directory path . Copy that path and paste it where it asked you for custom pass list and add the file name in the end of what you have pasted .. Email me if you still having issue .. (Bots.nightmare@gmail.com)

**Comment by yo712** [comment]
`home`

**Comment by BotKi11eR** [comment]
> `home`

?

**Comment by nethunaik96** [comment]
> > ![Screenshot_2022-12-09-10-38-05-93_84d3000e3f4017145260f7618db1d683.jpg](https://user-images.githubusercontent.com/120032131/206625154-1b4ebec3-c37a-4582-ac75-99ff935aff13.jpg)
> > 
> > Password list path ke 
> 
> Bro do you know what "Custom" means? You are supposed to have a txt file with some custom passwords you want to try on that account . And you need to give the location of that file in here . Go to the directory where your custom passwords file is . And type without brackets ( pwd ) . It will give you the directory path . Copy that path and paste it where it asked you for custom pass list and add the file name in the end of what you have pasted.

Yes we need to give(enter) the password  file's path(location where it is) to termux but I didn't understand how are we supposed to get that path example:  data/data/com.termux/
From where do we get this?
And what does directory mean?
And my password file is located in downloads of my internal storage

**Comment by Netdiebeste** [comment]
Hey what do you mean by a directory path?

**Comment by hackerboy178** [comment]
Bro now i get it, just don't select any option when you are on the select menu just type pwd then enter

**Comment by hackerboy178** [comment]
And if that's not help than just select back option then try

**Comment by kingvidar** [comment]
I don't understand

**Comment by Darsh-KillerTech** [comment]
![Screenshot_2024-03-11-11-05-51-59_84d3000e3f4017145260f7618db1d683.jpg](https://github.com/MrHacker-X/BruteX/assets/158742129/a91f6581-cb5b-4cca-84c6-507a09c66b53)

I got this error in the end, how can I resolve it??

**Comment by Yabi47** [comment]
/data/data/com.termux/files/home/pass.txt
